added titles to the docs pages (#109)
dappforce/subsocial-docs: docs/develop/sdk/posts/CreatePosts.md

title: Create and update Posts
displayed_sidebar: developSidebar
---

<head>

>   <title>Create and Update Posts on the Subsocial Blockchain | JS SDK Guide</title>

Reviewer: ```suggestion
  <title>Creating and Updating Posts With The Subsocial JS SDK</title>
```
